packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/entities/Event.java

```diff
@@ -30,34 +32,37 @@
 	@Index(columnList = "type"),
 	@Index(columnList = "value")
 })
-@EqualsAndHashCode(callSuper = true)
 public class Event extends PanacheEntityBase implements Serializable {
```

Comment:
Is `implements Serializable` needed? I remember you mentioning that it was not necessary